lib/index.js

```diff
@@ -12,7 +12,7 @@ const { apply } = require('./flags');
 module.exports = (cli) => {
   updateNotifier({ pkg }).notify();
 
-  process.env.WEBPACK_COMMAND = true;
+  process.env.WEBPACK_COMMAND = '1';
```

Comment:
So apparently the only valid types are strings. I'm not sure how these actually process when they are picked up but TIL!

Reply:
it's evaluated as `"true"` in process.env